rfcs/text/0049/entity.yml

```diff
@@ -0,0 +1,115 @@
+---
+# Entity field set schema definition
+name: entity
+title: Entity
+group: 2
+type: group
+short: Fields to describe various types of entities across IT environments.
+description: >
+  The entity fields provide a standardized way to represent and categorize
+  different types of components within an IT environment, including those
+  that don't have dedicated field sets in ECS. An entity represents a discrete,
+  identifiable component that can be described by a set of attributes and
+  maintains its identity over time.
+root: true
+fields:
+
+  - name: entity.id
+    level: core
+    type: keyword
+    short: Unique identifier for the entity.
+    description: >
+      A unique identifier for the entity. When multiple identifiers exist, this should be
+      the most stable and commonly used identifier that: 1) persists across the entity's
+      lifecycle, 2) ensures uniqueness within its scope, 3) is commonly used for queries
+      and correlation, and 4) is readily available in most observations (logs/events).
+      For entities with dedicated field sets (e.g., host, user), this value should match
+      the corresponding *.id field. Alternative identifiers (e.g., ARNs values in AWS, URLs)
+      can be preserved in entity.raw.
```

Comment:
> this value should match the corresponding *.id field. Alternative identifiers (e.g., ARNs values in AWS, URLs) can be preserved in entity.raw

Specially the ARN values in AWS seems to contradict what was previously suggested as guidance to entity ids. Am I misreading something?

Reply:
Nothing has changed there, I'll just make the last sentence more explicit, something like

> If the entity has multiple identifiers, these alternative identifiers can be preserved in entity.raw.*

Reply:
https://github.com/elastic/ecs/pull/2461/commits/f71cc784e02ca10bfb80eebe9d3a67edf5bd695f